exercises/practice/legacy-service-client/.meta/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "The classical introductory exercise. Just say \"Hello, World!\".",
+  "blurb": "Deprecated",
```

Comment:
Could you find the original blurb of the exercise? It would not matter for new students, but it would for older students.

Reply:
These didn't have a blurb because they haven't been updated since you created the config.json files in the V3 move.